contextConsumption/retrieve_entity_with_ldcontext_test.js

```diff
@@ -227,5 +292,29 @@ describe('Retrieve Entity. JSON-LD. @context ', () => {
     const response = await http.get(entitiesResource + entityId + '?attrs=P1', headers);
     assertRetrieved(response, expectedEntityNoAttr, JSON_LD);
   });
+
+  
+  it('should retrieve the entity but compaction shall use an alternative @context', async function() {
+    const headers = {
+      'Accept': 'application/ld+json',
+      // Observe that the provided @context will make the attribute not to match
```

Comment:
Not sure what you mean with this comment - wasn't the entity created/modified with alternativeContext.jsonld?
If there is no matching, as you state, how come you expect the entityAltCompacted?

Reply:
well spotted an old comment not needed!